src/Selenium2Library/keywords/_browsermanagement.py

```diff
@@ -507,6 +507,34 @@ def set_browser_implicit_wait(self, seconds):
         implicit_wait_in_secs = robot.utils.timestr_to_secs(seconds)
         self._current_browser().implicitly_wait(implicit_wait_in_secs)
 
+    def get_browser_info(self, type='object'):
+        """Gets information from webdriver on current browser.
+
+        By default returns an Object, otherwise if the argument `type`is different of 'object', returns
+        a Dictionary.
+
+        Examples of returned attributes are:
+        'rotatable','takesScreenshot', 'acceptSslCerts', 'cssSelectorsEnabled', 'javascriptEnabled',
+        'databaseEnabled', 'locationContextEnabled', 'platform', 'browserName', 'version',
+        'nativeEvents', 'applicationCacheEnabled', 'webStorageEnabled', 'handlesAlerts'
+
+        Examples:
```

Comment:
Could the example be more concrete. Explain that when this type of browser is opened then x contains y.